_posts/2023-11-16-virginia-beach-city/virginia-beach-city.Rmd

```diff
@@ -0,0 +1,229 @@
+
+## Analysis
+
+### Minority Gerrymandering: Virginia Beach City is Compliant with the Virginia Voting Rights Act
+
+The Virginia Voting Rights Act mandates that at-large local elections are banned if they dilute the voting power of minority populations. Instead, based on the percentage of the on the percentage of minorities in the citizen Voting Age Population and how cohesive minority voters are, minority opportunity districts would be created to protect minority voting power. Virginia Beach City has a cohesive democratic-leaning minority population that accounts for 32.8% of the citizen Voting Age Population. As there are 10 city council seats, three seats must be in MMDs.
+
+The minority voting age population plot is displayed below. Districts are sorted in order of lowest minority VAP percentage to highest, with blue or red denoting whether the district in any given plan has a Democratic or Republican tilt. The grey bars show the current ratified plan, showing that the current plan exceeds in creating three VAP districts. Notably, the enacted plan intentionally creates three solid majority minority districts, demonstrated by the gap between the 7th and 8th districts.
+
+![Figure 7: *Simulated Majority Minority District counts*](fig7minperfplot.png)
+
+We study the number of districts that are MMDs in any of the 200,000 plans. Total counts for MMDs in each simulation is displayed below:
+
+``` {.txt .R}
+  n_minority_perf     n
+1               0 95824
+2               1 88121
+3               2 13811
+4               3  2176
+5               4    68
+
+  n_black_perf      n
+1            0  56171
+2            1 143829
+```
+
+As simulations usually only create one MMD (\~88,000 simulations) or even none at all (\~96,000 simulations), it is clear that the enacted plan intentionally upholds the need for three MMDs based on the Virginia Voting Rights Act.
+
+### Partisan Gerrymandering: Low Competitiveness from Plans
+
+![Figure 8: *Democratic vote share by district in simulated runs. Black squares indicate the actual democratic vote share in the enacted plan, ordered by lowest to highest.*](fig8demrepsplit.png)
+
+## Summary of Simulations
+
+![](validation_20230903_2333.png)
+
+From the validation plot of our simulations, we see that our simulation ranges around the same level of compactness as the enacted plan. Simulated plans are diverse, but push against the generally acceptable boundary of a maximum 5% population deviation between districts and vary greatly in neighborhood splits. One observation is that the enacted plan packs minority voters into three districts, which was motivated by a substantially lower citizen minority voter population across Virginia Beach City. In effect, minorities in the three districts lie around a 51-53% citizen VAP majority.
```

Comment:
Good preliminary analysis of results. What broader research question do these findings answer?